doc/neps/nep-0037-array-module.rst

```diff
@@ -505,6 +514,10 @@ operators in Python, but it has two downsides:
 In contrast, importing a new library (e.g., ``import  dask.array as da``) with
 an API matching NumPy is entirely explicit. There is no overhead from dispatch
 or ambiguity about which implementation is being used.
+*FIXME: the above statement on overhead doesn't seem completely true, given
+that there is no explicit import but rather a ``get_array_module(some_array)``
+call for every public method or function in the consuming library (e.g.
+Scikit-learn) which also needs to do type-checking.*
```

Comment:
Wanted to get a second opinion on this one before trying to rephrase and/or benchmark.

Reply:
Well, I think the statements are all still true, but you could add that `get_array_module()` is also partially implicit since it tries to help libraries be implicit. However, I have to say that many library functions will probably do multiple API, so the overhead is potentially smaller.

Reply:
Updated by taking over @shoyer's description.